bitnami/zookeeper/Chart.yaml

```diff
@@ -21,4 +21,4 @@ name: zookeeper
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/zookeeper
   - https://zookeeper.apache.org/
-version: 11.0.2
+version: 11.0.3
```

Comment:
As you are adding a new parameter, please bump the version to a new minor (`11.1.0`)